R/gsplot-class.R

```diff
@@ -19,8 +23,12 @@ gsplot <- function(x = NULL, ...) UseMethod("gsplot")
 
 #' @rdname gsplot
 #' @export
-gsplot.default <- function(...,config.file=NA, theme=NA, frame.plot=TRUE) {
-  object <- gsplot(list(global=list('config'=list(frame.plot=frame.plot, 
+gsplot.default <- function(..., created=Sys.Date(), 
+                           gsplot.version=packageDescription("gsplot", fields = "Version"),
```

Comment:
better yet is 
```r
packageVersion(getPackageName())
```
so we don't need to modify it if we change the name of the package.